server/bin/pbench-audit-server.py

```diff
@@ -173,7 +185,7 @@ def verify_tar_dirs(ihier, tarball_dirs, tblist, controller):
 
 
 # 'ihier' is IncomingHierarchy class object
-def verify_incoming(ihier, verifylist):
+def verify_incoming(ihier, verifylist, hierarchy=False):
```

Comment:
What does the new 'hierarchy' parameter mean here? It doesn't appear to be used in this method.

Reply:
Yes it is not being used just some implementation loop hole.